src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2CommercialButtons.tsx

```diff
@@ -91,7 +91,6 @@ const ArtworkSidebar2CommerialButtons: React.FC<ArtworkSidebar2CommercialButtons
   }
 
   const handleCreateOrder = () => {
-    // console.log("I am here!")
```

Comment:
🫡

Reply:
oh, I didn't even notice this `console.log` 😅